Tests/SwaggerParserTests/SwaggerParserTests.swift

```diff
@@ -1,20 +1,129 @@
-
 class SwaggerParserTests: XCTestCase {
-    func testUber() {
-        do {
-            let swagger = try Swagger(JSONString: uberJSONString)
-            print(swagger)
-        } catch {
-            print(error)
+    var testFixtureFolder: URL!
+    
+    override func setUp() {
+        testFixtureFolder = URL(fileURLWithPath: #file).deletingLastPathComponent().appendingPathComponent("Fixtures")
+    }
+    
+    func testInitialization() {
+        let url = testFixtureFolder.appendingPathComponent("uber.json")
+        let jsonString = try! NSString(contentsOfFile: url.path, encoding: String.Encoding.utf8.rawValue) as String
+        let swagger = try! Swagger(JSONString: jsonString)
+        
+        XCTAssertEqual(swagger.host?.absoluteString, "api.uber.com")
+    }
+    
+    func testAllOfSupport() {
+        let url = testFixtureFolder.appendingPathComponent("test_all_of.json")
+        let jsonString = try! NSString(contentsOfFile: url.path, encoding: String.Encoding.utf8.rawValue) as String
+        let swagger = try! Swagger(JSONString: jsonString)
+        
+        guard
+            let baseDefinition = swagger.definitions.first(where: {$0.name == "TestAllOfBase"}),
+            case .object(let baseSchema) = baseDefinition.structure else {
+                XCTFail("TestAllOfBase is not an object schema."); return
         }
+        
+        SwaggerParserTests.validate(testAllOfBaseSchema: baseSchema)
+        swagger.definitions.validate(testAllOfChild: "TestAllOfFoo", withPropertyNames: ["foo"])
+        swagger.definitions.validate(testAllOfChild: "TestAllOfBar", withPropertyNames: ["bar"])
     }
+}
 
+/// MARK: Helper Functions
 
-    static var allTests = [
-        ("testUber", testUber),
-    ]
+fileprivate extension SwaggerParserTests {
+    /// Gets the base schema and child schema from a definition that defines an
+    /// `allOf` with one $ref (the base class) and one object schema.
+    class func getBaseAndChildSchemas(withDefinition definition: Structure<Schema>) -> (base: ObjectSchema?, child: ObjectSchema?) {
+        var base: ObjectSchema?
+        var child: ObjectSchema?
+        
+        if case .allOf(let schema) = definition.structure {
+            XCTAssertEqual(schema.schemas.count, 2)
+            
+            schema.schemas.forEach {
+                switch $0 {
+                case .object(let childSchema):
+                    child = childSchema
+                case .structure(let structure):
+                    guard case .object(let baseSchema) = structure.structure else {fatalError()}
+                    base = baseSchema
+                default: fatalError()
```

Comment:
I dislike putting statements on the same line as the colon. Also let's not fatalError() - how about throwing an error or doing XCTFail